src/SkprConfig.php

```diff
@@ -133,7 +223,10 @@ protected function convertToEnvVarName($key): string {
    * @return array
    *   Values.
    */
-  public function getAll(bool $environment_format = FALSE, string $filename = self::DEFAULT_FILENAME): array {
+  public function getAll(
+    bool $environment_format = FALSE,
+    string $filename = self::CONFIG_FILENAME
+  ): array {
```

Comment:
Personally not a fan of multiline function definitions

Reply:
Maybe we should add a code sniff rule for it?